.circleci/config.yml

```diff
@@ -257,24 +283,20 @@ commands:
       - run:
           name: Build Common Typescript Packages
           command: ./.circleci/build-ts-packages.sh
-      - save_cache:
-          name: Save Typescript Build
-          key: fxa-ts-build-01-<< pipeline.git.revision >>
+
+  save-workspace:
```

Comment:
See comment below for more info about approximate time savings, but this turns out to be a pretty big improvement and doesn't require a new cache entry.